x/authenticator/ante/ante.go

```diff
@@ -22,22 +21,18 @@ type AuthenticatorDecorator struct {
 	authenticatorKeeper *authenticatorkeeper.Keeper
 	accountKeeper       authante.AccountKeeper
 	sigModeHandler      authsigning.SignModeHandler
-	next                sdk.AnteHandler
```

Comment:
I think next should be passed to both classic and authenticator. Otherwise we are requiring the circuit breaker to be the last decorator. This may be in issue if we want any decorator to be used after authentication

Reply:
hmmm would prefer to do this in a subsequent PR, they way this flow work in the current implementation would be adding the new AnteHandler to the `authenticatorVerificationDecorator` and then the `classicSignatureVerificationDecorator` if it's needed there too.

I think this is a good point tho